### PR TITLE
Add clientInformation (alias of navigator) for all browsers

### PR DESCRIPTION
**Pull request by foolip**
Follow-up to https://github.com/mdn/browser-compat-data/pull/11690.

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -4150,27 +4168,69 @@
...
+                "version_added": true,
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "opera": [
+              {
+                "version_added": "3"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4185]:
```suggestion
                "version_added": "15",
```

Based on testing with http://mdn-bcd-collector.appspot.com/tests/api/Window/clientInformation

File: api/Window.json [hunk 2]
```diff
@@ -4150,27 +4168,69 @@
...
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "opera_android": [
+              {
+                "version_added": "10.1"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4194]:
```suggestion
                "version_added": "14",
```

Mirroring desktop.

File: api/Window.json [hunk 3]
```diff
@@ -4150,27 +4168,69 @@
...
-            },
-            "webview_android": {
-              "version_added": "1"
-            }
+            "ie": [
+              {
+                "version_added": "4"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4176]:
```suggestion
                "version_added": "≤6",
```

Based on testing.

File: api/Window.json [hunk 4]
```diff
@@ -4123,15 +4123,33 @@
...
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "edge": [
+              {
+                "version_added": "12"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4149]:
```suggestion
                "version_added": "12",
```

Because it was in IE.

File: api/Window.json [hunk 5]
```diff
@@ -4123,15 +4123,33 @@
...
-            },
-            "edge": {
-              "version_added": "12"
-            },
+            "chrome": [
+              {
+                "version_added": "1"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4131]:
```suggestion
                "version_added": "1",
```

Based on https://trac.webkit.org/changeset/1024/webkit (almost the beginning of WebKit)

File: api/Window.json [hunk 6]
```diff
@@ -4123,15 +4123,33 @@
...
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "18"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4140]:
```suggestion
                "version_added": "18",
```

Based on https://trac.webkit.org/changeset/1024/webkit (almost the beginning of WebKit)

File: api/Window.json [hunk 7]
```diff
@@ -4150,27 +4168,69 @@
...
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "safari": [
+              {
+                "version_added": "1"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4203]:
```suggestion
                "version_added": "1",
```

Based on https://trac.webkit.org/changeset/1024/webkit (almost the beginning of WebKit)

File: api/Window.json [hunk 8]
```diff
@@ -4150,27 +4168,69 @@
...
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "1"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4212]:
```suggestion
                "version_added": "1",
```

Based on https://trac.webkit.org/changeset/1024/webkit (almost the beginning of WebKit)

File: api/Window.json [hunk 9]
```diff
@@ -4150,27 +4168,69 @@
...
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "samsunginternet_android": [
+              {
+                "version_added": "1.0"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4221]:
```suggestion
                "version_added": "1.0",
```

Based on https://trac.webkit.org/changeset/1024/webkit (almost the beginning of WebKit)

File: api/Window.json [hunk 10]
```diff
@@ -4150,27 +4168,69 @@
...
+                "alternative_name": "clientInformation"
+              }
+            ],
+            "webview_android": [
+              {
+                "version_added": "1"
+              },
+              {
+                "version_added": true,
```
Comment (foolip) [inline api/Window.json:4230]:
```suggestion
                "version_added": "1",
```

Based on https://trac.webkit.org/changeset/1024/webkit (almost the beginning of WebKit)

**Review by vinyldarkscratch — APPROVED** [review]

**Pull request merged**